DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -84,10 +84,14 @@ object BlobSinker extends SinkOperatorFactory {
         blobStorageKey
       )
       timeNow = System.nanoTime()
+      AppInsightLogger.trackEvent(
+        ProductConstant.ProductRoot + "/WriteEventsToBlob",
+        Map("Timestamp" -> timeNow.toString, "OutputPath" -> outputPath),
```

Comment:
can we get the timetaken in a single value instead of logging start and stop? seems it would help metrics building.